blocks/hooks/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import createHooks from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import anchor from './anchor';
+
+const hooks = createHooks();
+
+const {
+	addAction,
```

Comment:
At this moment we use only:
- `addFilter`
- `removeAllFilters` (accessed directly from `@wordpress/hooks`)
- `applyFilters`

Reply:
I would have liked to simplify this to some pattern for exporting all available keys from `createHooks()`, but it's not supported by module exports.

https://stackoverflow.com/questions/29844074/es6-export-all-values-from-object

Would you suggest limiting this only to those that are currently used?

Reply:
Yes, I would take a similar approach to what we do for `wp.element`, where we cherry-pick only what is really essential to make things work. I'm in favor of starting with a very limited lits of exposed methods and extend it whenever we have a real-life use case.

Reply:
It's still not public API, so we can iterate later. Anyway, we should definitely revisit when working on documentation.